tests/Etc/ScopedComment.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Stancl\Tenancy\Tests\Etc;
+
+use Stancl\Tenancy\Database\Concerns\BelongsToPrimaryModel;
+
+class ScopedComment extends Comment
+{
+    use BelongsToPrimaryModel;
```

Comment:
This class uses single-db scoping logic? If so, then probably add comments to these models so that it's clear how the setup works.

Reply:
Answered [here](https://github.com/archtechx/tenancy/pull/1105#discussion_r1200367668)

> add comments to these models so that it's clear how the setup works

Right